internal/archiver/archiver_test.go

```diff
@@ -959,6 +959,14 @@ func TestArchiverSaveDirIncremental(t *testing.T) {
 	}
 }
 
+// Equals fails the test if exp is not equal to act.
+func bothNonZeroOrNot(tb testing.TB, exp, act uint64) {
+	if (exp == 0 && act != 0) || (exp != 0 && act == 0) {
+		_, file, line, _ := runtime.Caller(1)
+		tb.Fatalf("\033[31m%s:%d:\n\n\texp: %#v\n\n\tgot: %#v\033[39m\n\n", filepath.Base(file), line, exp, act)
+	}
+}
+
```

Comment:
Isn't this just Equals(exp > 0, act > 0)?

Reply:
Basically yes, however, the error message contains the full values of exp and act and not just true/false.

Reply:
Ok. I think a more obvious function name would be `notBothZero`. The comment could then be "notBothZero fails the test if only one of exp, act is zero" (the current comment has a grammatical error).

Reply:
Thanks for the suggestions.